imsv-docs-docusaurus/openapi/endpoints/login/models/login-init-request.yaml

```diff
@@ -0,0 +1,53 @@
+type: object
+required:
+  - clientApplicationId
+  - scopes
+properties:
+  loginMethod:
+    description: The login method to use.
+    type: string
+    enum:
+      - siwe
```

Comment:
i dont think we should expose all these `loginMethods` yet since we don't support them.

Reply:
would it be better to attach Authentication Guide link for supported method instead of using enum, so that it's more maintainable?